.github/workflows/ci.yml

```diff
@@ -204,6 +205,55 @@ jobs:
           GHA_RUN_DC: false
         run: mvn ${{ env.MAVEN_CLI_OPTS }} verify -pl 'hxinsight-extension,e2e-test' -am -DskipUnitTests -DskipIntegrationTests -Dalfresco-platform.version=${{ matrix.repoVersion }}
 
+  run_e2e_test_other_supported_java:
+    name: "Run e2e tests using ACS ${{ matrix.repoVersion }} on Java 11"
+    runs-on: ubuntu-latest
+    needs:
+      - pre_commit
+    if: >
+      github.event_name != 'pull_request' &&
+      (github.ref_name == 'master' || startsWith(github.ref_name, 'release/')) &&
+      !contains(github.event.head_commit.message, '[skip build]') &&
+      !contains(github.event.head_commit.message, '[skip tests]') &&
+      !contains(github.event.head_commit.message, '[skip e2es]')
+    strategy:
+      fail-fast: false
+      matrix:
+        repoVersion: [ 7.3.2, 7.4.2 ]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: Alfresco/alfresco-build-tools/.github/actions/free-hosted-runner-disk-space@v5.33.0
+      - name: "Login to Quay.io"
+        uses: docker/login-action@v3.2.0
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v5.33.0
+      - name: "Build application"
+        if: env.JAVA_VERSION_SUPPORTED != env.JAVA_VERSION
```

Comment:
maybe JAVA_11? and btw - will it always be false?

Reply:
I don't get the idea of these checks😂

Reply:
It depends if in future we will have the need to again support some older version of Java then used in project 🤔

Reply:
After our support for ACS 7.4 will finish we can then fix `JAVA_VERSION_SUPPORTED=17`, but maybe this job could be then removed 🤔 and this ifs won't be necessary 🤔

Reply:
I wanted to came up with kind of a universal solution for CI, and disable whole job if connector and alfresco java version were the same, but GHA `job.if` and `job.name` statements doesn't have access to `env` context 🙄 What is funny `job.steps.if` statement has access to `env`

Reply:
it sounds unlikely that we'll need to add support for other java versions🤔 @mpichura wdyt? I would keep it as simple as possible and just hardcode these versions

Reply:
Thanks to @damianujma hint I was able to fully parametrize this job, so it behaves as expected - run only for multiple supported Java versions; in different case don't even start